packages/nextjs/scaffold.config.ts

```diff
@@ -20,7 +20,7 @@ const scaffoldConfig = {
   // You can get your own at https://dashboard.alchemyapi.io
   // It's recommended to store it in an env variable:
   // .env.local for local testing, and in the Vercel/system env config for live apps.
-  alchemyApiKey: process.env.NEXT_PUBLIC_ALCHEMY_API_KEY || "oKxs-03sij-U_N0iOlrSsZFr29-IqbuF",
+  alchemyApiKey: process.env.NEXT_PUBLIC_ALCHEMY_API_KEY || "",
```

Comment:
Remove the "this is ours Alchemy`s default API KEY"

Another thing: Should we hint something about this in the README? 

Here maybe:
![image](https://github.com/user-attachments/assets/de667708-26d7-4b15-a22b-4345538e4941)

- You can add your Alchemy API Key in `scaffold.config.ts` if you want more reliability in your RPC requests. 

(or something similar)

Reply:
Ohh yeah missed thanks updated!